components/private_channel/rust/ffi/config.gni

```diff
@@ -0,0 +1,75 @@
+declare_args() {
```

Comment:
this stuff is no longer relevant, please see https://github.com/brave/brave-core/pull/6408

Reply:
- [x] remove `declare_args()`